observer/dispatcher.go

```diff
@@ -29,23 +25,20 @@ func (d *Dispatcher) dispatch(event Event) {
 		Action: event.Tx.Type,
 		Result: event.Tx,
 	}
-	txJson, err := json.Marshal(action)
-	if err != nil {
-		logger.Panic(err)
-	}
-
 	webhook := event.Subscription.Webhook
 	logParams := logger.Params{
 		"webhook": webhook,
 		"coin":    event.Subscription.Coin,
 		"txID":    event.Tx.ID,
 	}
-	go d.postWebhook(webhook, txJson, logParams)
+	go d.postWebhook(webhook, action, logParams)
 	logger.Info("Dispatching webhooks...", logger.Params{"webhook": webhook}, logParams)
 }
 
-func (d *Dispatcher) postWebhook(hook string, data []byte, logParams logger.Params) {
-	_, err := d.Client.Post(hook, "application/json", bytes.NewReader(data))
+func (d *Dispatcher) postWebhook(hook string, data interface{}, logParams logger.Params) {
+	client := blockatlas.InitClient(hook)
+	client.Headers["Content-Type"] = "application/json"
+	err := client.Post(nil, "", data)
```

Comment:
What is `nil` here?

Reply:
because the function doesn't need the return object